cfgov/v1/jinja2/v1/includes/organisms/tables/case-docket.html

```diff
@@ -0,0 +1,44 @@
+<h2 class="h5">Case docket</h2>
+
+<table class="o-table u-w100pct">
+    <thead>
+        <tr>
+            <th scope="col">Document</th>
+            <th scope="col">Date filed</th>
+            <th scope="col">Short description</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for event in value.events -%}
+        <tr>
+            <td>
+                {% if event.document -%}
+                    <a href="{{ event.document.url }}">
+                {%- endif -%}
+                {{ event.index }}
+                {%- if event.document -%}
+                    </a>
+                {%- endif %}
```

Comment:
We have this pattern a few places and I wish it wasn't so cluttered. I'm not sure if this is any better (feels obfuscating), but for your consideration, you can do this in two lines. Maybe it inspires a better way:

```
{% set link_markup = ['<a href="' ~ event.document.url ~ '">','</a>'] if event.document else ['',''] %}
{{ link_markup | join(event.index) | safe }}
```

Reply:
Thanks for the suggestion @anselmbradford. Personally I find the `{% if %} ... {% else %}` more readable in the template, and using Jinja tags with dashes like `-%}` ensures that things can still render as one line in the HTML.

Also: we should try to reduce unnecessary uses of `| safe`! Here there should be no reason to use it unless there's some character in the URL that needs to be escaped.